config

```diff
@@ -0,0 +1,66 @@
+####################
+# General settings #
+####################
+
+# OO_PROVIDER
+#
+# Selects the cloud provider to use
+# Choices: gce, aws
+# Default: gce
+
+#export OO_PROVIDER=aws
+
+################
+# GCE settings #
+################
+
+################
+# AWS settings #
+################
+
+# Credentials
+#
+# AWS_ACCESS_KEY_ID
+# AWS_SECRET_ACCESS_KEY
+
+#source ~/aws-credentials
```

Comment:
One of the changes that I was working tries to pull from either the ~/.awscred file or the aws cli default configs...  let me go ahead and get that change separated out and in an isolated PR.